addons/pos_sale/static/tests/tours/pos_sale_tour.js

```diff
@@ -376,3 +376,25 @@ registry.category("web_tour.tours").add("PoSDownPaymentFixedTax", {
             }),
         ].flat(),
 });
+
+registry.category("web_tour.tours").add("PosSettleOrderWithLot", {
+    test: true,
+    url: "/pos/ui",
+    steps: () =>
+        [
+            Dialog.confirm("Open session"),
+            PosSale.selectNthOrder(1),
+            {
+                content: `Choose to settle the order`,
+                trigger: `.modal:not(.o_inactive_modal) .selection-item:contains('Settle the order')`,
+                in_modal: false,
+                run: "click",
+            },
+            {
```

Comment:
I believe you could add an  option to the settleNtOrder utils something like ´loadSN´ that would add a stept. So that you can use it in your case.